src/config.rs

```diff
@@ -32,7 +32,7 @@ fn parse_cli_args() -> Result<CliArgs, getopts::Fail> {
     let args: Vec<String> = std::env::args().collect();
 
     let mut opts = getopts::Options::new();
-    opts.optopt("", "url", "URL to load on start", "URL");
```

Comment:
I want the argument to be simply `url`. Could this be fixed by somewhere else?

Reply:
Ok. I renamed `initial-url` to `url`